spec/factories/developer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :developer do
+    sequence :name do |n|
+      "Developer-#{n}"
+    end
+  end
+end
```

Comment:
The developer seems awfully skiny, why not give it more attributes? Also this factory is not annotated for some reason